app/static/js/social_links.js

```diff
@@ -0,0 +1,37 @@
+$(window).on("load", () => {
+	$(".social-del").click(toggleSocialInputDelete);
+	$(".social-add").click(e => {
+		e.preventDefault();
+		
+		const form_index = $(`#id_${FORM_PREFIX}-TOTAL_FORMS`).val();
+		const new_form = $($("#social_form_empty").html().replace(/__prefix__/g, form_index));
+		$(".social-inputs").append(new_form);
+		$(`#id_${FORM_PREFIX}-TOTAL_FORMS`).val((form_index/1)+1);
```

Comment:
```suggestion
		const form_index = parseInt($(`#id_${FORM_PREFIX}-TOTAL_FORMS`).val());
		const new_form = $($("#social_form_empty").html().replace(/__prefix__/g, form_index));
		$(".social-inputs").append(new_form);
		$(`#id_${FORM_PREFIX}-TOTAL_FORMS`).val(form_index+1);
```
For consistency reasons